modules/javafx.controls/src/test/java/test/javafx/scene/control/TableViewKeyInputTest.java

```diff
@@ -191,6 +192,19 @@ private boolean isAnchor(int row, int col) {
         assertTrue(sm.isSelected(1));
     }
 
+    @Test
+    public void testEnterOnFocusedRowDoesNotThrowNP() {
```

Comment:
The usual abbreviation for NullPointerException is `NPE` rather than `NP`, so I recommend updating the test method names and comments accordingly.

Reply:
You are right, my bad. Fixed it. :)